### DISCUSSION TITLE
Which is the last version without user accounts?

### DISCUSSION THREAD

**Discussion by BananaAcid** [open-webui/open-webui#518] [General]
I unfortunately changed the ollama ip on the container settings in portainer, and i had the :lastet tag on the image name.

Now it has all that user management stuff (no verification email/no email-domain restriction) and server side saving of chat protocols. I want the simple version back, but I am not sure what the correct version is.

Any idea?

**Comment by tjbck** [comment]
You're probably looking for https://github.com/ollama-webui/ollama-webui-lite !
